engine/engine.go

```diff
@@ -150,7 +151,14 @@ func loadConfigWithSettings(settings *Settings, flagSet map[string]bool) (*confi
 func validateSettings(b *Engine, s *Settings, flagSet map[string]bool) {
 	b.Settings = *s
 
-	b.Settings.EnableDataHistoryManager = (flagSet["datahistorymanager"] && b.Settings.EnableDatabaseManager) || b.Config.DataHistoryManager.Enabled
+	b.Settings.EnableDataHistoryManager = (flagSet["datahistorymanager"] &&
+		b.Settings.EnableDatabaseManager) ||
+		b.Config.DataHistoryManager.Enabled
```

Comment:
I find the old way more readable, but if you insist, I recommend this because of the brackets:
```suggestion
	b.Settings.EnableDataHistoryManager = (flagSet["datahistorymanager"] &&  	b.Settings.EnableDatabaseManager) ||
		b.Config.DataHistoryManager.Enabled
```